src/components/Modals/Send/views/Address.tsx

```diff
@@ -78,9 +86,36 @@ export const Address = () => {
 
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
+  const isInAddressBookFilter = useMemo(
+    () => ({ accountAddress: address, chainId: asset?.chainId }),
+    [address, asset?.chainId],
+  )
+  const isInAddressBook = useAppSelector(state =>
+    selectIsAddressInAddressBook(state, isInAddressBookFilter),
+  )
+
+  const internalAccountIdFilter = useMemo(
+    () => ({ accountAddress: address, chainId: asset?.chainId }),
+    [address, asset?.chainId],
+  )
+  const internalAccountId = useAppSelector(state =>
+    selectInternalAccountIdByAddress(state, internalAccountIdFilter),
+  )
+
   const supportsENS = asset?.chainId === ethChainId // We only support ENS resolution on ETH mainnet
   const addressError = get(errors, `${SendFormFields.Input}.message`, null)
 
+  useEffect(() => {
+    setShowSaveButton(
+      Boolean(
+        (!isInAddressBook || internalAccountId) &&
```

Comment:
also why do we show save for internal accounts? Should already work OOTB nah?

Reply:
It will be in the next branch, disabled it for internalAccount in [757a7dd](https://github.com/shapeshift/web/pull/10803/commits/757a7ddd6631de0719b78e63f9b94a87b7039543)